.github/workflows/flow-temp.yml

```diff
@@ -30,18 +30,18 @@ jobs:
   #     platform: "amazonlinux:2"
   #     test-config: "TEST=test:test_with_tls"
 
-  # test-debian:
-  #   needs: [docs-only, get-latest-redis-tag]
-  #   if: needs.docs-only.outputs.only-docs-changed == 'false'
-  #   uses: ./.github/workflows/flow-linux-platforms.yml
-  #   secrets: inherit
-  #   with:
-  #     redis-ref: ${{ needs.get-latest-redis-tag.outputs.tag }}
-  #     platform: "debian:bullseye"
-
-  test-macos:
-    uses: ./.github/workflows/flow-macos.yml
+  test-debian:
+    needs: [docs-only, get-latest-redis-tag]
+    if: needs.docs-only.outputs.only-docs-changed == 'false'
+    uses: ./.github/workflows/flow-linux-platforms.yml
     secrets: inherit
     with:
-      coordinator: true
-      test-config: QUICK=1
+      redis-ref: ${{ needs.get-latest-redis-tag.outputs.tag }}
+      platform: "debian:bookworm"
```

Comment:
Comment out once you see all is good

Reply:
OK. Changes in flow-temp.yml were reverted.